.gitmodules

```diff
@@ -30,3 +30,7 @@
   path = sorc/upp.fd
   url = https://github.com/NOAA-EMC/UPP.git
   ignore = dirty
+[submodule "sorc/jcb"]
+  path = sorc/jcb
+  url = https://github.com/noaa-emc/jcb
```

Comment:
Are the yaml's in the jcb repo under `configuration/` used?  
If so, we will need to think about how to move them to `parm/` for operational acceptance.
Ideally, we would install `jcb` as a python package, and the yamls would be part of global-workflow/gdasapp.

Reply:
I was tying to play around with submodules to see if jcb could not clone its submodules since in the context of global-workflow they are indeed not needed. But I was not successful. The submodule of jcb (jcb-gdas) is also a submodule of GDASapp and that would be where the configuration files that global-workflow actually comes in and gets linked to `parm/`. Perhaps we can brainstorm a better way of doing this so that jcb-gdas is not a submodule of jcb. It's like that primarily to facilitate easy testing of jcb in a standalone context but perhaps it could be done with pip dependency or some other way that I'm not thinking about.

Reply:
jcb-algorithms repo has been established, moving the generic JEDI-algortithm YAMLs to the gdas parm. Now there should be no YAML files in non-parm directories that are used anywhere.